### PR TITLE
Feature/email address

### PR DESCRIPTION
**Pull request by tc-tibo**

### CODE REVIEW

File: CDTS Blazor/Utilities/SessionState.cs [hunk 1]
```diff
@@ -13,6 +13,10 @@ public class SessionState
     {
 
         public UploadDocumentPageModel UploadDocumentPage { get; set; }
+        public SessionState()
+        {
+            UploadDocumentPage = new UploadDocumentPageModel();
```
Comment (tc-chris) [inline CDTS Blazor/Utilities/SessionState.cs:18]: Why are you doing this? It messes up with the logic

File: CDTS Blazor/Data/Services/MtoaEmailService.cs [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using MTOA.DomainObjects.DTO;
+using System.Net.Http;
+using System.Net.Http.Formatting;
+using System.Diagnostics;namespace CDNApplication.Data.Services
+{
+    public class MtoaEmailService
+    {
+        private string api_key = "d00bfe90183b42cc8e78627c20db8d64";
+        private string jwt = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1laWQiOiJDU0YiLCJ1bmlxdWVfbmFtZSI6IlNlYWZhcmVyIENlcnRpZmljYXRpb24gKENTRikiLCJyb2xlIjoiVXNlciIsIkNvcnJlbGF0aW9uS2V5IjoiNjI3YzIwZGI4ZDY0IiwibmJmIjoxNTk5NTk5OTUyLCJleHAiOjE2NjI2NzE5NTIsImlhdCI6MTU5OTU5OTk1MiwiaXNzIjoiVHJhbnNwb3J0IENhbmFkYSIsImF1ZCI6IlRyYW5zcG9ydCBDYW5hZGEifQ.NpFgM8AgoX0Rq0vMKVF_crLuZPxr1KJHEw-DpvlEwQc";
```
[the remark this answers is not in the document]
Comment (tc-tibo) [reply CDTS Blazor/Data/Services/MtoaEmailService.cs:16]: Yes. AB#46120 if you want to grab it.

File: CDTS Blazor/Utilities/SessionState.cs (hunk 1 above)
Comment (tc-tibo) [reply CDTS Blazor/Utilities/SessionState.cs:18]: Pls. run a git-blame to see the history on it and check with that person.

File: CDTS Blazor/Data/Services/MtoaEmailService.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (tc-chris) [reply CDTS Blazor/Data/Services/MtoaEmailService.cs:16]: Done

**Review by tropical-turtle — APPROVED** [review]
Reviewed by TNT mob.

**Pull request merged**